Firmware not compiling using yotta under Windows 10 (ninja)

Dear,

The actual master do not compile using Yotta.

System : 
- commit : bfaa52a2bb1c934a89202d44bb50172405a5b1bb 
- OS : Windows 10
- Yotta  : v023

Commands : 

```
$ git clone https://github.com/bbcmicrobit/micropython
$ cd micropython
$ yt target bbc-microbit-classic-gcc-nosd
$ yt up
$ yt build
```

Output : 

> (the long command)
> D:/Workspace/yotta/micropython/yotta_modules/ble-nrf51822/source/nRF5xGap.cpp: In member function 'virtual ble_error_t nRF5xGap::setWhitelist(const Gap::Whitelist_t&)':
> D:/Workspace/yotta/micropython/yotta_modules/ble-nrf51822/source/nRF5xGap.cpp:689:59: error: 'BLEProtocol::AddressType_t' is not a class or namespace
>          if (whitelistIn.addresses[i].type == BLEProtocol::AddressType_t::RANDOM_PRIVATE_NON_RESOLVABLE) {
>                                                            ^
> D:/Workspace/yotta/micropython/yotta_modules/ble-nrf51822/source/nRF5xGap.cpp: In member function 'ble_error_t nRF5xGap::generateStackWhitelist(ble_gap_whitelist_t&)':
> D:/Workspace/yotta/micropython/yotta_modules/ble-nrf51822/source/nRF5xGap.cpp:906:61: error: 'BLEProtocol::AddressType_t' is not a class or namespace
>          if (whitelistAddresses[i].addr_type == BLEProtocol::AddressType_t::RANDOM_PRIVATE_RESOLVABLE) {
>                                                              ^
> ninja: build stopped: subcommand failed.
> error: command ['ninja'] failed

I am experiencing the same issue on Linux (Ubuntu 16.04.1):

Output:

> /microbit/micropython/yotta_modules/ble-nrf51822/source/nRF5xGap.cpp: In member function 'virtual ble_error_t nRF5xGap::setWhitelist(const Gap::Whitelist_t&)':
/microbit/micropython/yotta_modules/ble-nrf51822/source/nRF5xGap.cpp:689:59: error: 'BLEProtocol::AddressType_t' is not a class or namespace
         if (whitelistIn.addresses[i].type == BLEProtocol::AddressType_t::RANDOM_PRIVATE_NON_RESOLVABLE) {
                                                           ^
/microbit/micropython/yotta_modules/ble-nrf51822/source/nRF5xGap.cpp: In member function 'ble_error_t nRF5xGap::generateStackWhitelist(ble_gap_whitelist_t&)':
/microbit/micropython/yotta_modules/ble-nrf51822/source/nRF5xGap.cpp:906:61: error: 'BLEProtocol::AddressType_t' is not a class or namespace
         if (whitelistAddresses[i].addr_type == BLEProtocol::AddressType_t::RANDOM_PRIVATE_RESOLVABLE) {
                                                             ^

--- a/module.json
+++ b/module.json
@@ -8,7 +8,7 @@
   "homepage": "http://micropython.org",
   "dependencies":{
     "mbed-classic": "~0.0.4",
-    "microbit-dal":"lancaster-university/microbit-dal#v1.4.19"
+    "microbit-dal":"lancaster-university/microbit-dal#v1.4.19b"
   },
   "extraIncludes":[
     "inc",

